app/domain/authentication/authn_k8s/kubectl_exec.rb

```diff
@@ -153,23 +116,29 @@ def on_message(msg, ws_client)
           ws_client.close
         end
       end
-      
+
       def on_close
         @channel_closed = true
         @logger.debug("Pod #{@pod_name} : channel closed")
       end
 
       def on_error(err)
         @channel_closed = true
-        @logger.debug("Pod #{@pod_name} error : #{err.inspect}")
-        @message_log.save_error_string(err.inspect)
+
+        error_info = err.inspect
+        @logger.debug("Pod #{@pod_name} error : #{error_info}")
+        @message_log.save_error_string(error_info)
       end
 
       private
 
+      def kubeclient
+        @kubeclient ||= @k8s_object_lookup.kubectl_client
```

Comment:
What is `@k8s_object_lookup` resolving to...? I don't see it defined anywhere else in the class.

Reply:
It's in the dependencies: https://github.com/cyberark/conjur/blob/3e2b06c183da59a177f6c06c055c8b6bab8001d1/app/domain/authentication/authn_k8s/kubectl_exec.rb#L58